### DISCUSSION TITLE
Can I call APIs with this?

### DISCUSSION THREAD

**Discussion by wistfulglen** [squidfunk/mkdocs-material#5031] [Q&A]
Still very new to this whole thing, so apologies for the possibly obvious question. I'm curious if it's possible for Material to connect and call APIs?

**Comment by squidfunk** [comment]
If the API you want to call is callable from the browser, yes, definitely. [We're already doing it](https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/javascripts/components/source/facts/github/index.ts) when rendering the number of GitHub stars and forks. You can add your call via [additional JavaScript](https://squidfunk.github.io/mkdocs-material/customization/#additional-javascript), maybe something like:

`docs/javascripts/api.js`:

``` js
fetch("https://api.github.com/repos/squidfunk/mkdocs-material")
  .then(response => response.json())
  .then(console.log)
  .catch(console.error)
```

`mkdocs.yml`:

``` yaml
extra_javascript:
  - javascripts/api.js
```
